build_docker

```diff
@@ -21,8 +21,8 @@
 # SOFTWARE.
 
 if [[ ! -f src/tools/certificat.crt ]]; then
```

Comment:
I think we shall remove all of this IF section. My goal is to try to avoid to have any file creation within the current directory when the build occurs. People are going to issue the build command within there git tree. If we create file into it this will mess up there next pull request or some of them might be pushing there temporary certificates.